src/main/java/org/openstreetmap/atlas/geography/atlas/items/complex/aoi/ComplexAOIRelation.java

```diff
@@ -0,0 +1,112 @@
+package org.openstreetmap.atlas.geography.atlas.items.complex.aoi;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.openstreetmap.atlas.geography.MultiPolygon;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasEntity;
+import org.openstreetmap.atlas.geography.atlas.items.Relation;
+import org.openstreetmap.atlas.geography.atlas.items.complex.ComplexEntity;
+import org.openstreetmap.atlas.geography.atlas.items.complex.RelationOrAreaToMultiPolygonConverter;
+import org.openstreetmap.atlas.tags.filters.TaggableFilter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Complex Entity for AOI Relations. AOI relations are those that are {@link MultiPolygon} and has
+ * AOI tags in it. AOI tags are checked against the {@link TaggableFilter} passed as an argument or
+ * to the default {@link TaggableFilter} of AOI tags if the tags are not explicitly specified.
+ *
+ * @author sayas01
+ */
+public final class ComplexAOIRelation extends ComplexEntity
+{
+    private static final Logger logger = LoggerFactory.getLogger(ComplexAOIRelation.class);
+    private static final RelationOrAreaToMultiPolygonConverter RELATION_TO_MULTI_POLYGON_CONVERTER = new RelationOrAreaToMultiPolygonConverter();
+    // The default AreasOfInterest(AOI) tags
+    private static final TaggableFilter AOI_TAG_FILTER = TaggableFilter.forDefinition(
```

Comment:
I would consider turning this into a json resource, as I can see the list of aois growing substantially.